modules/installation-creating-aws-worker.adoc

```diff
@@ -41,19 +41,19 @@ template requires:
 [
   {
     "ParameterKey": "InfrastructureName", <1>
-    "ParameterValue": "mycluster-vw9j6" <2>
+    "ParameterValue": "mycluster-<random_string>" <2>
   },
   {
     "ParameterKey": "RhcosAmi", <3>
-    "ParameterValue": "ami-07f604c5f18a1e7a9" <4>
+    "ParameterValue": "ami-<random_string>" <4>
   },
   {
     "ParameterKey": "WorkerSubnet", <5>
     "ParameterValue": "yes" <6>
```

Comment:
Sorry ... found another problem. This parameter key should be called PrivateSubnet. The Value should be sg-<random> string. The parameter should be modified in the template yaml also @wking

Reply:
https://github.com/openshift/openshift-docs/pull/15086